README.md

```diff
@@ -46,8 +46,72 @@ chances for property name collision in Archaius.
 
 Using this pattern, the properties stored in Cerberus will treated as just another set of properties in addition to 
 existing sources like property files. All properties can then be accessed using the 
-DynamicPropertyFactory.getInstance().getXXX methods.
+DynamicPropertyFactory.getInstance().getXXX methods. This can be achieve with either nonpolling or polling configuration.
 
+##### Nonpolling configuration (Recommended)
+
+Most applications do not need to poll Cerberus for changes.  If secrets change, the instances can simply be restarted
+or redeployed to pick up new values.
+The code shown below is a sample implementation of AbstractModule that instantiates a ConcurrentMapConfiguration and 
+adds it to the existing ConfigurationManager.
+
+``` java
+    public class CerberusArchaiusModule extends AbstractModule {
+        private final Logger logger = LoggerFactory.getLogger(getClass());
+    
+        private static final String SECRETS_PATH_PROP_NAME = "cerberus.config.path";
+        private static final String SECRETS_DEFAULT_VAULT_PATH = "app/cerberus-demo/config";
+    
+        @Override
+        protected void configure() {
+            /*
+             * First let's look up the path for where to read in Cerberus.
+             * This examples show us attempting to source it from an Archaius property, but defaulting if not found.
+             */
+            final String vaultPath = DynamicPropertyFactory.getInstance()
+                    .getStringProperty(SECRETS_PATH_PROP_NAME, SECRETS_DEFAULT_VAULT_PATH)
+                    .get();
+    
+            /*
+             * So long as we've got a path, let's pull the config into Archaius from Cerberus.
+             */
+            if (vaultPath != null && !vaultPath.isEmpty()) {
+                logger.info("Adding Cerberus as Configuration source. Vault Path = " + vaultPath);
+    
+                /*
+                 * Create a configuration source passing in a Cerberus client using the ArchaiusCerberusClientFactory and
+                 * the path we looked up above.  This factory will attempt to resolve cerberus configuration detailts,
+                 * like the URL and token from Archaius properties.
+                 */
+                final PolledConfigurationSource polledConfigurationSource = new NamespacedCerberusConfigurationSource(
```

Comment:
Are you sure we need a `PolledConfigurationSource` when we aren't polling?

Did you dig through the Archiaus library to confirm?

Reply:
Added some code to do things properly